### PR TITLE
docs(astro): Add multi-framework example for $signInStore and $signUpStore

### PR DESCRIPTION
**Pull request by wobsoriano**
> [!IMPORTANT]
> 🔎 Previews:
>
> - https://clerk.com/docs/pr/1435/references/astro/sign-in-store
> - https://clerk.com/docs/pr/1435/references/astro/sign-up-store

### This PR:

- As one of Astro's key features is its ability to seamlessly integrate components from various frameworks, this PR demonstrates the use of @clerk/astro's `$signInStore` and `$signUpStore` with both Vue and Svelte components.

### CODE REVIEW

File: docs/references/astro/sign-in-store.mdx [hunk 1]
```diff
@@ -5,34 +5,66 @@ description: Clerk's $signInStore nanostore provides a convenient way to access
 
 # `$signInStore`
 
-The `$signInStore` store provides a convenient way to access the [`SignIn`](/docs/references/javascript/sign-in/sign-in){{ target: '_blank' }} object, which allows you to check the current state of a sign-in. This is also useful for creating a custom sign-in flow.
+The `$signInStore` store provides a convenient way to access the [`SignIn`](https://clerk.com/docs/references/javascript/sign-in/sign-in) object, which allows you to check the current state of a sign-in. This is also useful for creating a custom sign-in flow.
```
Comment (alexisintech) [inline docs/references/astro/sign-in-store.mdx:8]:
```suggestion
The `$signInStore` store provides a convenient way to access the [`SignIn`](/docs/references/javascript/sign-in/sign-in){{ target: '_blank' }} object, which allows you to check the current state of a sign-in. This is also useful for creating a custom sign-in flow.
```

File: docs/references/astro/sign-in-store.mdx [hunk 2]
```diff
@@ -5,34 +5,66 @@ description: Clerk's $signInStore nanostore provides a convenient way to access
...
 
 ### Check the current state of a sign-in
 
 The following example demonstrates how to use the `$signInStore` store to check the current state of a sign-in.
 
-```tsx {{ filename: 'sign-in-step.tsx' }}
-import { useStore } from '@nanostores/react'
-import { $signInStore } from '@clerk/astro/client'
+<CodeBlockTabs options={['React', 'Vue', 'Svelte']}>
+  ```tsx {{ prettier: false, filename: 'sign-in-step.tsx' }}
```
Comment (alexisintech) [inline docs/references/astro/sign-in-store.mdx:17]: Is there a reason we want to turn off prettier for the React code block?

```suggestion
  ```tsx {{ filename: 'sign-in-step.tsx' }}
```

File: docs/references/astro/sign-in-store.mdx [hunk 3]
```diff
@@ -5,34 +5,66 @@ description: Clerk's $signInStore nanostore provides a convenient way to access
...
+  {/if}
+  ```
+</CodeBlockTabs>
 
 The possible values for the `status` property of the `SignIn` resource are listed [here](/docs/references/javascript/sign-in/sign-in#properties).
 
 ### Create a custom sign-in flow
 
-The `$signInStore` store can also be used to build fully custom sign-in flows, if Clerk's pre-built components don't meet your specific needs or if you require more control over the authentication flow. Different sign-in flows include email and password, email and phone codes, email links, and multifactor (MFA). To learn more about using the `$signInStore` store to create custom flows, check out the [custom flow guides](/docs/custom-flows/overview).
+The `$signInStore` store can also be used to build fully custom sign-in flows, if Clerk's pre-built components don't meet your specific needs or if you require more control over the authentication flow. Different sign-in flows include email and password, email and phone codes, email links, and multifactor (MFA). To learn more about using the `$signInStore` store to create custom flows, check out the [custom flow guides](/docs/custom-flows/overview)
```
Comment (alexisintech) [inline docs/references/astro/sign-in-store.mdx:70]:
```suggestion
The `$signInStore` store can also be used to build fully custom sign-in flows, if Clerk's pre-built components don't meet your specific needs or if you require more control over the authentication flow. Different sign-in flows include email and password, email and phone codes, email links, and multifactor (MFA). To learn more about using the `$signInStore` store to create custom flows, check out the [custom flow guides.](/docs/custom-flows/overview)
```

File: docs/references/astro/sign-up-store.mdx [hunk 4]
```diff
@@ -5,34 +5,66 @@ description: Clerk's $signUpStore nanostore provides a convenient way to access
 
 # `$signUpStore`
 
-The `$signUpStore` store provides a convenient way to access the [`SignUp`](/docs/references/javascript/sign-up/sign-up){{ target: '_blank' }} object, which allows you to check the current state of a sign-up. This is also useful for creating a custom sign-up flow.
+The `$signUpStore` store provides a convenient way to access the [`SignUp`](https://clerk.com/docs/references/javascript/sign-up/sign-up) object, which allows you to check the current state of a sign-up. This is also useful for creating a custom sign-up flow.
```
Comment (alexisintech) [inline docs/references/astro/sign-up-store.mdx:8]:
```suggestion
The `$signUpStore` store provides a convenient way to access the [`SignUp`][`SignUp`](/docs/references/javascript/sign-up/sign-up){{ target: '_blank' }} object, which allows you to check the current state of a sign-up. This is also useful for creating a custom sign-up flow.
```

File: docs/references/astro/sign-up-store.mdx [hunk 5]
```diff
@@ -5,34 +5,66 @@ description: Clerk's $signUpStore nanostore provides a convenient way to access
...
 
 ### Check the current state of a sign-up
 
 The following example demonstrates how to use the `$signUpStore` store to access the `SignUp` object and check the current state of a sign-up.
 
-```tsx {{ filename: 'sign-up-step.tsx' }}
-import { useStore } from '@nanostores/react'
-import { $signUpStore } from '@clerk/astro/client'
+<CodeBlockTabs options={['React', 'Vue', 'Svelte']}>
+  ```tsx {{ prettier: false, filename: 'sign-up-step.tsx' }}
```
Comment (alexisintech) [inline docs/references/astro/sign-up-store.mdx:17]: Same question about formatting the React code block

```suggestion
  ```tsx {{ filename: 'sign-up-step.tsx' }}
```

File: docs/references/astro/sign-up-store.mdx [hunk 6]
```diff
@@ -5,34 +5,66 @@ description: Clerk's $signUpStore nanostore provides a convenient way to access
...
+  {#if $signUp === undefined}
+      <!-- Add logic to handle loading state -->
+  {:else}
+      <div>The current sign-up attempt status is {$signUp.status}.</div>
+  {/if}
+  ```
+</CodeBlockTabs>
 
-The possible values for the `status` property of the `SignUp` resource are listed [here](/docs/references/javascript/sign-up/sign-up#properties){{ target: '_blank' }}.
+The possible values for the `status` property of the `SignUp` resource are listed [here](https://clerk.com/docs/references/javascript/sign-up/sign-up#properties).
```
Comment (alexisintech) [inline docs/references/astro/sign-up-store.mdx:66]:
```suggestion
The possible values for the `status` property of the `SignUp` resource are listed [here](/docs/references/javascript/sign-up/sign-up#properties){{ target: '_blank' }}.
```

File: docs/references/astro/sign-up-store.mdx [hunk 7]
```diff
@@ -5,34 +5,66 @@ description: Clerk's $signUpStore nanostore provides a convenient way to access
...
 
 ### Create a custom sign-up flow
 
-The `$signUpStore` store can also be used to build fully custom sign-up flows, if Clerk's pre-built components don't meet your specific needs or if you require more control over the authentication flow. Different sign-up flows include email and password, email and phone codes, email links, and multifactor (MFA). To learn more about using the `$signUpStore` store to create custom flows, check out the [custom flow guides](/docs/custom-flows/overview).
+The `$signUpStore` store can also be used to build fully custom sign-up flows, if Clerk's pre-built components don't meet your specific needs or if you require more control over the authentication flow. Different sign-up flows include email and password, email and phone codes, email links, and multifactor (MFA). To learn more about using the `$signUpStore` store to create custom flows, check out the [custom flow guides](/docs/custom-flows/overview)
```
Comment (alexisintech) [inline docs/references/astro/sign-up-store.mdx:70]:
```suggestion
The `$signUpStore` store can also be used to build fully custom sign-up flows, if Clerk's pre-built components don't meet your specific needs or if you require more control over the authentication flow. Different sign-up flows include email and password, email and phone codes, email links, and multifactor (MFA). To learn more about using the `$signUpStore` store to create custom flows, check out the [custom flow guides.](/docs/custom-flows/overview)
```

**Review by alexisintech — APPROVED** [review]

File: docs/references/astro/sign-up-store.mdx [hunk 8]
```diff
@@ -5,34 +5,66 @@ description: Clerk's $signUpStore nanostore provides a convenient way to access
 
 # `$signUpStore`
 
-The `$signUpStore` store provides a convenient way to access the [`SignUp`](/docs/references/javascript/sign-up/sign-up){{ target: '_blank' }} object, which allows you to check the current state of a sign-up. This is also useful for creating a custom sign-up flow.
+The `$signUpStore` store provides a convenient way to access the \[`SignUp`][`SignUp`](/docs/references/javascript/sign-up/sign-up){{ target: '_blank' }} object, which allows you to check the current state of a sign-up. This is also useful for creating a custom sign-up flow.
```
Comment (bradlc) [inline docs/references/astro/sign-up-store.mdx:8]:
```suggestion
The `$signUpStore` store provides a convenient way to access the [`SignUp`](/docs/references/javascript/sign-up/sign-up){{ target: '_blank' }} object, which allows you to check the current state of a sign-up. This is also useful for creating a custom sign-up flow.
```

**Pull request merged**